docs/pages/build-reference/ios-capabilities.md

```diff
@@ -24,43 +26,62 @@ If a capability is enabled for your app remotely, but not present in the native
+| <YesIcon /> | Media Device Discovery                            | `com.apple.developer.media-device-discovery-extension`                     |
+| <YesIcon /> | Multipath                                         | `com.apple.developer.networking.multipath`                                 |
+| <YesIcon /> | NFC Tag Reading                                   | `com.apple.developer.nfc.readersession.formats`                            |
+| <YesIcon /> | Network Extensions                                | `com.apple.developer.networking.networkextension`                          |
+| <YesIcon /> | On Demand Install Capable for App Clip Extensions | `com.apple.developer.on-demand-install-capable`                            |
+| <YesIcon /> | Personal VPN                                      | `com.apple.developer.networking.vpn.api`                                   |
+| <YesIcon /> | Push Notifications                                | `aps-environment`                                                          |
+| <YesIcon /> | Push to Talk                                      | `com.apple.developer.push-to-talk`                                         |
+| <YesIcon /> | Recalibrate Estimates                             | `com.apple.developer.healthkit.recalibrate-estimates`                      |
+| <YesIcon /> | Shared with You                                   | `com.apple.developer.shared-with-you`                                      |
+| <YesIcon /> | Sign In with Apple                                | `com.apple.developer.applesignin`                                          |
+| <YesIcon /> | SiriKit                                           | `com.apple.developer.siri`                                                 |
+| <YesIcon /> | System Extension                                  | `com.apple.developer.system-extension.install`                             |
+| <YesIcon /> | TV Services                                       | `com.apple.developer.user-management`                                      |
+| <YesIcon /> | Time Sensitive Notifications                      | `com.apple.developer.usernotifications.time-sensitive`                     |
+| <YesIcon /> | Wallet                                            | `com.apple.developer.pass-type-identifiers`                                |
+| <YesIcon /> | WeatherKit                                        | `com.apple.developer.weatherkit`                                           |
+| <YesIcon /> | Wireless Accessory Configuration                  | `com.apple.external-accessory.wireless-configuration`                      |
+| <YesIcon /> | iCloud                                            | `com.apple.developer.icloud-container-identifiers`                         |
+| <NoIcon />  | HLS Interstitial Previews                         | ???                                                                        |
```

Comment:
Instead of `???` can we have something different? For example, how about specifying "Not Supported"? I think it's good if we can be explicit about it. Ignore this comment if you think `???` is better. I am unfamiliar with the terminology "HLS Interstitial Previews", so I could be wrong about this comment.